src/content/peerconnection/endtoend-encryption/js/main.js

```diff
@@ -79,6 +75,41 @@ function start() {
       });
 }
 
+// We use a Worker to do the encryption and decryption.
+// See
+//   https://developer.mozilla.org/en-US/docs/Web/API/Worker
+// for basic concepts.
+const worker = new Worker('./js/worker.js');
+function setupSenderTransform(sender) {
+  const senderStreams = sender.track.kind === 'video' ? sender.createEncodedVideoStreams() : sender.createEncodedAudioStreams();
+  // Instead of creating the transform stream here, we do a postMessage to the worker. The first
+  // argument is an object defined by us, the sceond a list of variables that will be transferred to
+  // the worker. See
+  //   https://developer.mozilla.org/en-US/docs/Web/API/Worker/postMessage
+  /*
+  const transformStream = new TransformStream({
```

Comment:
Missing: If you don't want to use a worker, you can instead do:

As written, it looks like a random block of commented-out code.